### PR TITLE
Fixing/improving commands.

### PR DESCRIPTION
**Pull request by artursouza**

### CODE REVIEW

File: tutorial/README.md [hunk 1]
```diff
@@ -20,10 +20,14 @@ After completing the getting started section you should be connected to an AKS c
 Clone this repository locally using whatever technique you prefer. Example:
 
 ```sh
-git clone git@github.com/azure/azure-oam-solution.git
+git clone git@github.com:Azure/azure-oam-solution.git
```
Comment (rynowak) [inline tutorial/README.md:23]: This literally has no effect right? 

I'm surprised that Azure hasn't renamed their org to lower case.... The rest of `microsoft` and `dotnet` did :)

File: tutorial/README.md [hunk 2]
```diff
@@ -20,10 +20,14 @@ After completing the getting started section you should be connected to an AKS c
...
 ```
 
-This repository contains code you will use during the tutorial as well as a solution you can use to check your work. You might find it convenient to use the `./tutorial/flight-tracker` directory as your working directory for the bulk of the tutorial.
```
Comment (rynowak) [inline tutorial/README.md:26]: This should still be here.

File: tutorial/README.md (hunk 1 above)
Comment (artursouza) [reply tutorial/README.md:23]: Casing has no effect, the `:` does make a difference.

**Review by rynowak — APPROVED** [review]
:shipit: after restoring the deleted paragraph. Other changes look good, do whatever you want about the casing of the URL 😆

File: tutorial/README.md (hunk 2 above)
Comment (artursouza) [reply tutorial/README.md:26]: All the tutorial works from the `tutorial` folder.

**Comment by artursouza** [comment]
The casing was a nit thing but adding `:` made the command work.

File: tutorial/README.md (hunk 2 above)
Comment (rynowak) [reply tutorial/README.md:26]: The explanation of why we asked to clone the repo should still be here.

Comment (artursouza) [reply tutorial/README.md:26]: This is not an explanation of why to close the repo. It is just telling the user which directory he might find convenient to use. Now, this tutorial already handles that in the commands. Moving to a different directory can break some commands.

Comment (rynowak) [reply tutorial/README.md:26]: Ok fair point. Agreed on taking this out.

**Pull request merged**